test/modules/TestResultCoordinator/Reports/DirectMethod/LongHaul/DirectMethodLongHaulReportGenerator.cs

```diff
@@ -72,14 +74,22 @@ public async Task<ITestResultReport> CreateReportAsync()
                         senderSuccesses++;
                         break;
                     default:
```

Comment:
From `test/modules/TestResultCoordinator/Reports/DirectMethod/LongHaul/DirectMethodLongHaulReport.cs ` comment: ` // fail if we find anything that is not a 200 or 0 (most notably, 500's)`. it shouldn't be possible to have `200` in the `other` right? This switch case still think `200` should be part of the dict.

Reply:
I don't think 200 will ever show up in the dict, since it will go to the case 200: senderSuccesses++ lines